src/drawers/arcOutlineDrawer.ts

```diff
@@ -0,0 +1,21 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Drawers {
+
+  export class ArcOutline extends Drawer {
+
+    constructor(dataset: Dataset) {
+      super(dataset);
+      this._className = "arc outline";
+      this._svgElementName = "path";
+    }
+
+    protected _applyDefaultAttributes(selection: d3.Selection<any>) {
+      super._applyDefaultAttributes(selection);
+      selection.style("fill", "none");
+      selection.style("stroke-linejoin", "round");
```

Comment:
Do we want the user to have control over this? If so, we should define it in **plottable.css** instead of here.